app/web/routes.py

```diff
@@ -74,9 +93,37 @@ def new_incident(current_user):
             impact=form.incident_impact.data,
             start_date=form.incident_start.data,
             components=incident_components,
+            system=False,
         )
         db.session.add(new_incident)
         db.session.commit()
+
+        current_app.logger.debug(
+            f"{new_incident} opened by {get_user_string(current_user)}"
+        )
+
+        messages_from = []
+
+        for inc in active_incidents:
+            messages_to = []
+            for comp in incident_components:
+                if comp in inc.components:
+                    messages_to.append(
```

Comment:
I'm not sure that it is correct to use a specific attribute key value directly in the code, in routes. Make sure that this is processed on the frontend, and is sent to the route as an argument.